enterprise_access/apps/api/v1/views/customer_billing.py

```diff
@@ -510,11 +514,8 @@ def get_serializer_class(self):
         STRIPE_EVENT_SUMMARY_READ_PERMISSION,
         fn=stripe_event_summary_permission_detail_fn,
     )
-    def list(self, request, *args, uuid=None, **kwargs):
+    def list(self, request, *args, **kwargs):
```

Comment:
We're just going to have a list endpoint, so we don't need a specific parameter for an object to retrieve.